### DISCUSSION TITLE
ANSSI 2.0 and changes of the rules impacted by it

### DISCUSSION THREAD

**Discussion by rumch-se** [ComplianceAsCode/content#10674] [General]
Hello @ssato 

I think that you created anssi profile covering version 2.0. If this is not correct, please include in this discussion the contributor who created it.  
 
Today I found that some of the rules described in this profile keep in their rule.yml file the old anssi recommendation - BP28(RX), where X is a number of recommendation.  
For example check these and anssi code used their references section 
linux_os/guide/services/ntp/chronyd_configure_pool_and_server/rule.yml
linux_os/guide/services/ntp/chronyd_specify_remote_server/rule.yml
linux_os/guide/services/ntp/package_chrony_installed/rule.yml
linux_os/guide/services/ntp/service_chronyd_or_ntpd_enabled/rule.yml
linux_os/guide/services/ntp/service_ntp_enabled/rule.yml

Is there a PR which covers this remapping and sets correct codes in all rules which are described into anssi profile?

Have a nice day
Rumen

**Comment by vojtapolasek** [comment]
The correct person is @yuumasato

**Comment by rumch-se** [comment]
Hello @yuumasato 

Please check my post. There are a lot of rules, which still have the old anssi codes. The new anssi profile was defined in controls directory, but the rules were not updated. For example:
in the rule mount_option_home_nosuid we have anssi: BP28(R12), but in the anssi profile the rule is a part of recommendation R28.  
Today I will fix this.

Have a nice day
Rumen

**Reply by yuumasato** [reply]
Hi, @rumch-se

It seems we forgot to update the reference in some of the rules.
Thank you for checking them.
